src/components/organisms/top/top.html

```diff
@@ -0,0 +1,18 @@
+<div class="top">
+  <div class="ima">
+    <p class="cafe">
+      そうだ。<br />
+      カフェに行こう。
+    </p>
+    <img src="../../../img/top.jpg" class="assets" alt="..." />
+  </div>
+  <div class="tops">
```

Comment:
topsの意味がよくわからないので、top-contentとかの方がわかりやすいかも
理由はcssとか見ててtop?tops?ってなったからです
```suggestion
  <div class="top-content">
```